README.md

```diff
@@ -1 +1,10 @@
-# helm-charts
\ No newline at end of file
+# Helm Charts
+
+## Update a chart
+Enter the folder of the chart you want to update, then:
+
+`helm lint .`
+
+`helm dependecy update .`
+
+`helm push . colvin`
```

Comment:
Maybe it would be okay to put the version tag in the push?
`helm push . colvin --version 1.0.2`

Reply:
isn't it better to rely on the version set in the chart?

Reply:
You're right  ;)